README.md

```diff
@@ -4,33 +4,39 @@
 
 [![Coverage Status](https://coveralls.io/repos/github/zooniverse/Panoptes-Front-End/badge.svg)](https://coveralls.io/github/zooniverse/Panoptes-Front-End)
 
-## Development
+## Getting started
 
-Requires Node.js. It's recommended you manage your Node installations with **nvm**.
+### With Docker
 
-`npm install` installs dependencies.
+To avoid having to install Node.js or any other dependencies, you can run
+everything with Docker and Docker Compose.
 
-`npm start` builds and runs the site locally.
+- `docker-compose build` will build a local Docker image and run `npm install`. Run this whenever you
+change dependencies in `package.json`.
 
-`npm run stage` builds and optimizes the site, and then deploys it to <https://current-git-branch-name.pfe-preview.zooniverse.org>.
+- `docker-compose run --service-ports dev npm start` builds and runs the site locally on port 3735.
 
-For testing with production data, you can add `env=production` to your development url, e.g. `localhost:3735/projects?env=production`. Note that it is removed on every page refresh.
+- `docker-compose run dev npm test` runs the tests.
 
-All the good stuff is in **./app**. Start at **./app/main.cjsx**
+- `docker-compose run dev npm run stage` builds and optimizes the site, and then deploys it to <https://current-git-branch-name.pfe-preview.zooniverse.org>.
 
-While editing, do your best to follow style and architecture conventions already used by the project, unless you have a reason not to. **If in doubt, ask.**
+### With Node.js
 
-### Development with Docker
+Make sure you have Node 8 and `npm` 5 or greater. It's recommended you manage your Node installations with **nvm**.
 
-To avoid having to install Node.js or any other dependencies, you can also run
-everything with Docker and Docker Compose.
+- `npm install` installs dependencies.
 
-`docker-compose build` will build a local Docker image and run `npm install`. Run this whenever you
-change dependencies in `package.json`.
+- `npm start` builds and runs the site locally.
 
-`docker-compose run --service-ports dev npm start` builds and runs the site locally on port 3735.
+- `npm run stage` builds and optimizes the site, and then deploys it to <https://current-git-branch-name.pfe-preview.zooniverse.org>.
 
-`docker-compose run dev npm run stage` builds and optimizes the site, and then deploys it to <https://current-git-branch-name.pfe-preview.zooniverse.org>.
+### Development
+
+For testing with production data, you can add `env=production` to your development url, e.g. `localhost:3735/projects?env=production`. Note that it is removed on every page refresh.
+
+All the good stuff is in **./app**. Start at **./app/main.cjsx**
+
+While editing, do your best to follow style and architecture conventions already used by the project, unless you have a reason not to. **If in doubt, ask.**
```

Comment:
This instruction really isn't helpful at all to beginners. Should we delete it and ask people to check their contributions with `eslint` instead?

Reply:
Would having people use `eslint` cover everything that you were trying to express with the existing text? As a new contributor here, I think having both instructions are useful.

Reply:
Yeah, for the javascript we could have a line that says use eslint. For the coffeescript, though we do not have linting setup. Ideally no one should be editing the coffeescript components unless they're being converted, but realistically sometimes we have to do quick bug fixes on those old components.

Reply:
My problem with this text is that there isn't really a convention for the style and architecture of the code in this repo. I think figuring out where a new component should go can be a bit of a struggle for beginners. Saying "follow our lead, unless you don't want to" isn't helpful.

Reply:
If we have conventions for style and architecture, this paragraph should explicitly describe them for beginners, rather than hinting that they exist. That's what's bugging me about this text.

Reply:
Yeah unfortunately we have a mix of styles going on. I think we should suggest to beginners to use what we've been doing in the monorepo? That way any new code or any coffeescript to javascript files will be following that and maybe will ease the migration to the rewrite?

Reply:
That's a good idea. I'll add some text about `eslint` too. Something along these lines:
> We use `eslint` with a modified version of the AirBnB rules to keep our JavaScript code style consistent. Please lint your code using the `.eslintrc` file contained in this repo.

Reply:
I've added a note about eslint and a link to the monorepo as an example of how we organise components.